src/pkg/k8s/configmap.go

```diff
@@ -0,0 +1,59 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ReplaceConfigmap deletes and recreates a configmap
```

Comment:
Many of these comments are going to ding on the godot rule if you ran pre-commit on them...  Recommend running this though pre-commit when you are done.